utilities/verify-db-migration.sh

```diff
@@ -37,19 +37,20 @@ echo "MIGRATION_SQL_FILE: $MIGRATION_SQL_FILE"
 
 exit_if_binary_not_installed "pg_dump"
 
+export POSTGRESQL_DATABASE="postgres"
```

Comment:
As above, if the environment variable is already set by the user calling script, we should use the existing value, otherwise we should default to `postgres`